documentation/src/main/asciidoc/topics/ref_rest_protobuf.adoc

```diff
@@ -0,0 +1,82 @@
+[id='rest_v2_create_protobuf']
+= Creating Protobuf Schemas
+Create Protobuf schemas across {brandname} clusters with `POST` requests that
+include the content of a protobuf file in the payload.
+
+[source,options="nowrap",subs=attributes+]
+----
+POST /rest/v2/protobuf/{schemaName}
+----
+
+If the schema already exists, {brandname} returns `CONFLICT`. If the schema is
+not valid, {brandname} stores the schema and returns the error in the response body.
+
+{brandname} responds with the schema name and any errors.
+
+[source,json,options="nowrap",subs=attributes+]
+----
+include::json_examples/rest_protobuf_create_update_schema_response.json[]
+----
+
+* `name` is the name of the Protobuf schema.
+* `error` is `null` for valid Protobuf schemas. If {brandname} cannot successfully validate the schema, it returns errors.
+
+[id='rest_v2_get_protobuf']
+= Reading Protobuf Schemas
+Retrieve Protobuf schema from {brandname} with `GET` requests.
+
+[source,options="nowrap",subs=attributes+]
+----
+GET /rest/v2/protobuf/{schemaName}
+----
+
+[id='rest_v2_update_protobuf']
+= Updating Protobuf Schemas
+Modify Protobuf schemas with `PUT` requests that include the content of a
+protobuf file in the payload.
+
+[source,options="nowrap",subs=attributes+]
+----
+PUT /rest/v2/protobuf/{schemaName}
+----
+
+If the schema already exists, {brandname} returns `CONFLICT`. If the schema is
```

Comment:
PUT will not return CONFLICT: it will create if the schema does not exist or replace it otherwise

Reply:
Updating is not creating, so no